doc/hardware/emulator/bus_emulators.rst

```diff
@@ -144,6 +144,33 @@ Zephyr includes the following emulators:
 * MSPI emulator driver, allowing drivers to be connected to an emulator so that
   tests can be performed without access to the real hardware.
 
+I2C Emulation features
+----------------------
+
+In the binding of the I2C emulated bus, there's a custom property for address
+based forwarding. Given the following devicetree node:
+
+.. code-block::
+
+   i2c0: i2c@100 {
+     status = "okay";
+     compatible = "zephyr,i2c-emul-controller";
+     clock-frequency = <I2C_BITRATE_STANDARD>;
+     #address-cells = <1>;
+     #size-cells = <0>;
+     #forward-cells = <1>;
+     reg = <0x100 4>;
+     forwards = <&i2c1 0x20>;
```

Comment:
Nit - it would be helpful to demonstrate forwarding 2 addresses to the same bus.

Reply:
Done